ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/ListUserRightsWithGivenAdminUserIdAuthIT.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox.auth
+
+import com.daml.ledger.api.v1.admin.user_management_service._
+import io.grpc.{Status, StatusRuntimeException}
+
+import java.util.UUID
+import scala.concurrent.Future
+
+class ListUserRightsWithGivenAdminUserIdAuthIT
+    extends AdminServiceCallAuthTests
+    with UserManagementAuth {
+
+  override def serviceCallName: String = "UserManagementService#ListUserRights(given-user-id)"
+
+  // admin and idp admin users are allowed to specify a user-id other than their own for which to retrieve a user
+  override def serviceCall(context: ServiceCallContext): Future[Any] = {
+    val testId = UUID.randomUUID().toString
+
+    def getRights(userId: String): Future[ListUserRightsResponse] =
+      stub(context.token).listUserRights(ListUserRightsRequest(userId))
```

Comment:
idp field not populated, is that intentional?

Reply:
this class is removed, the original class is fixed